net.sonic_pi.SonicPi.json

```diff
@@ -150,14 +161,42 @@
                 }
             ]
         },
+        {
+            "name": "cmake",
```

Comment:
why do you need cmake? it's in the runtime.

Reply:
it's sure overkill to have to compile this, but `vcpkg` requires this version (the one in the runtime is the previous one).